### PR TITLE
Improve match of snatched NZB items in snatchSelection using size

### PR DESCRIPTION
**Pull request by fernandog**
Fixes the issue where N releases names from same provider (with same release name) were marked as SNATCHED but user had only one SNATCH

@duramato as we talked

TODO:

- [x] Uncomment minor version increment

image

### CODE REVIEW

**Comment by fernandog** [comment]
@duramato can you test it too?  it will need size now to "match"

**Comment by fernandog** [comment]
@duramato still waiting

File: medusa/history.py [hunk 1]
```diff
@@ -26,7 +26,7 @@
 
 
 def _logHistoryItem(action, showid, season, episode, quality, resource,
-                    provider, version=-1, proper_tags='', manually_searched=False, info_hash=None, resource_size=None):
+                    provider, version=-1, proper_tags='', manually_searched=False, info_hash=None, resource_size=-1):
```
Comment (labrys) [inline medusa/history.py]: Default should be `None` any for any values that are not determined.

File: medusa/server/web/home/handler.py [hunk 2]
```diff
@@ -1259,6 +1262,7 @@ def titler(x):
                 i['status'], i['quality'] = Quality.split_composite_status(i['action'])
                 i['action_date'] = sbdatetime.sbfdatetime(datetime.strptime(str(i['date']), History.date_format), show_seconds=True)
                 i['resource_file'] = os.path.basename(i['resource'])
+                i['resource_size'] = pretty_file_size(i['resource_size']) if i['resource_size'] > -1 else 'N/A'
```
Comment (labrys) [inline medusa/server/web/home/handler.py]: `i['resource_size'] = pretty_file_size(i['resource_size']) if i['resource_size'] else 'N/A'`

File: medusa/databases/main_db.py [hunk 3]
```diff
@@ -559,3 +559,22 @@ def execute(self):
...
+        """
+        Test if the version is at least 44.6
+        """
+        return self.connection.version >= (44, 6)
+
+    def execute(self):
+        backupDatabase(self.connection.version)
+
+        log.info(u"Adding column resource_size in history")
+        if not self.hasColumn("history", "resource_size"):
```
Comment (labrys) [inline medusa/databases/main_db.py]: Column name should be simply `size` for consistency with other columns... for example we have `provider` not `resource_provider` and `date` not `resource_date`.

File: medusa/history.py (hunk 1 above)
Comment (fernandog) [inline medusa/history.py]: size in medusa db and code is default to -1

File: medusa/databases/main_db.py (hunk 3 above)
Comment (duramato) [inline medusa/databases/main_db.py]: imo anything only for a release should be `resource_*`, to be descriptive for anyone checking the db

**Pull request merged**